alphagov/paas-cf
# [#163418605] Add Russell to merge users and CF admins

What
----

See title...

How to review
-------------

* Verify that this is adding Russell's key
* Verify I've got his email correct.

Who can review
--------------

Not me.
